pkg/controller/core/queuedworkload_controller.go

```diff
@@ -80,13 +83,13 @@ func (r *QueuedWorkloadReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 
 	status := workloadStatus(&wl)
 	if status == pending && !r.queues.QueueForWorkloadExists(&wl) {
-		err := workload.UpdateWorkloadStatusIfChanged(ctx, r.client, &wl, kueue.QueuedWorkloadAdmitted, corev1.ConditionFalse,
+		err := workload.UpdateWorkloadStatusIfChanged(ctx, r.client, &wl, r.recorder, kueue.QueuedWorkloadAdmitted, corev1.ConditionFalse,
```

Comment:
This was not an scheduling attempt. I don't think we need an Event in this case. The status should be enough.